### ISSUE TITLE
string-interpolation in ~w sigil

### ISSUE THREAD

**Issue by zhyu** [elixir-lang/emacs-elixir#263]
string-interpolation has been recognized as comment.
<img width="411" alt="2015-10-02 17 49 15" src="https://cloud.githubusercontent.com/assets/1728523/10242819/abe9e452-692e-11e5-94ee-d38157755b8a.png">

Not sure if this is related to #206

**Comment by syohex** [comment]
To change syntax property of hash mark of `#{}` can fix this issue. Patch is here. However I don't know this is correct way.

```diff
diff --git a/elixir-mode.el b/elixir-mode.el
index 8a9a80f..0f0e59b 100644
--- a/elixir-mode.el
+++ b/elixir-mode.el
@@ -256,6 +256,7 @@ is used to limit the scan."
 (defun elixir-syntax-propertize-interpolation ()
   (let* ((beg (match-beginning 0))
          (context (save-excursion (save-match-data (syntax-ppss beg)))))
+    (put-text-property beg (1+ beg) 'syntax-table (string-to-syntax "w"))
     (put-text-property beg (1+ beg) 'elixir-interpolation
                        (cons (nth 3 context) (match-data)))))
 

```

![elixir](https://cloud.githubusercontent.com/assets/554281/10244742/7a6d446a-693c-11e5-913a-4a6b963c0744.png)

**Comment by tonini** [comment]
Thanks, I'll have a look at this. :+1:

**Comment by zhyu** [comment]
It seems to be fixed in #264. Thanks, @syohex !
